__tests__/app.test.js

```diff
@@ -51,7 +51,8 @@ describe('GET /api/articles/:article_id', () => {
                 topic: expect.any(String),
                 created_at: expect.any(String),
                 votes: expect.any(Number),
-                article_img_url: expect.any(String)
+                article_img_url: expect.any(String),
+                comment_count: expect.any(Number)
```

Comment:
You could _expand_ your test suite rather than _modify_ it, by adding a new test for the comment_count property. As you're using the matcher `toMatchObject` the old test would still pass